core/src/main/java/hudson/model/UpdateSite.java

```diff
@@ -180,7 +180,6 @@ public long getDataTimestamp() {
         }
     }
 
-    @Restricted(NoExternalUse.class)
     public @Nonnull FormValidation updateDirectlyNow(boolean signatureCheck) throws IOException {
```

Comment:
```suggestion
    /**
     * Forces an update the data file from the configured URL, irrespective of the last time the data was retrieved.
     * @param signatureCheck whether to enforce the signature (may be off only for testing!)
     * @return A {@code FormValidation} indicating the if the update metadata was successfully downloaded from the configured update site
     * @since TODO
     * @throws IOException if there was an error downloading or saving the file.
     */
    public @Nonnull FormValidation updateDirectlyNow(boolean signatureCheck) throws IOException {
```